src/code/z_camera_data.c

```diff
@@ -35,10 +35,36 @@ s16 sOREGInit[] = {
 s16 sOREGInitCnt = 53;
 
 s16 sPREGInit[] = {
-    -20, 200, 300, 10, 12, 10, 35, 60, 60, 3, 0, -40, 20, 25, 45, -5, 15, 15, 20, 0, 0, 0, 6, 60, 30, 0, 5,
-};
-
-s16 sPREGInitCnt = 27;
+    -20, // R_CAM_DATA(CAM_DATA_Y_OFFSET)
```

Comment:
The CamData Regs is where this data specifically ends up, so having it there just helped with understanding the code. But I think having the `sCamDataRegsInit` named should be enough and it can be taken out now